plugins/filter/gcp_secret_filters.py

```diff
@@ -0,0 +1,60 @@
+# (c) 2019, Eric Anderson <eric.sysmin@gmail.com>
```

Comment:
I think this copyright is incorrect? seems like copy-pasted code from a separate file.

It should reflect the author of the code.